src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_BORDER = 5;
+    private static final int BORDER = 101;
+    private int randomFigureIndex;
```

Comment:
The field 'randomFigureIndex' should be a local variable inside the 'getRandomFigure()' method since it's not used outside of that context.